webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,15 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand")
+    .on("input", function() {
+      $(this).css({"overflow":"hidden", "height": "scrollHeight+px"});
```

Comment:
The string `"scrollHeight+px"` won't do what you want, that's just an invalid string as far as CSS is concerned. Did you try to run this code?

Try: `"height": this.scrollHeight + "px"`